spec/unit/plugins/darwin/cpu_spec.rb

```diff
@@ -78,7 +78,7 @@
 CTL
 
     allow(@plugin).to receive(:collect_os).and_return(:darwin)
-    allow(@plugin).to receive(:shell_out).with("sysctl -a").and_return(mock_shell_out(0, @stdout, ""))
+    allow(@plugin).to receive(:shell_out).with("sysctl cpu machdep").and_return(mock_shell_out(0, @stdout, ""))
```

Comment:
The code runs `sysctl hw` and the test expects `sysctl cpu` - looks like the source of the failures